From 44535d17ab082c59075e975ddffddde99942b9ff Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Wed, 26 Dec 2018 14:08:21 -0700
Subject: [PATCH 3/6] msm8974-common: sepolicy: Eliminate qemu_hw_mainkeys_prop
 entries

* Specific definition of this is dropped from qcom/sepolicy-legacy

Change-Id: I429abf7dddd2de4443349366b932149f30b87206
---
 sepolicy/common/priv_app.te | 1 -
 1 file changed, 1 deletion(-)

diff --git a/sepolicy/common/priv_app.te b/sepolicy/common/priv_app.te
index fe2dc8b..7bc944e 100644
--- a/sepolicy/common/priv_app.te
+++ b/sepolicy/common/priv_app.te
@@ -1,5 +1,4 @@
 get_prop(priv_app, camera_prop)
-get_prop(priv_app, qemu_hw_mainkeys_prop)
 
 allow priv_app device:dir r_dir_perms;
 allow priv_app proc_interrupts:file r_file_perms;
-- 
2.17.1

